code/game/objects/items/weapons/shields.dm

```diff
@@ -36,6 +36,32 @@
 	else
 		..()
 
+/obj/item/weapon/shield/riot/buckler
+	name = "buckler"
+	desc = "A small wooden shield. Its surface area is small, but it's still somewhat effective."
+	icon_state = "buckler"
+	w_class = 3.0
```

Comment:
@unid15 that's exactly why it shouldn't be there.

Same for trailing returns, for example.